meetings/eng/2021.md

```diff
@@ -1,5 +1,130 @@
 # 2021 Meetings
 
+## 2021-07-02
+
+### Meeting roles
+
+- **Meeting facilitator**: @choldgraf 
+- **Meeting recorder**: @damianavila
+- **Meeting timekeepr**: @choldgraf
+
+### Welcome to the Team Meeting
+
+Hello!
+
+If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+- name / GitHub handle
+- Sarah / @sgibson91
+- Damian / @damianavila 
+- Yuvi / @yuvipanda
+- Erik / @consideratio
+- Chris / @choldgraf 
+
+### Follow up items
+
+- [x] Chris sends the Binder cross stitch to his mom!
+- [x] Revisit times for the deliverable and monthly meetings (there are conflicts with other meetings)
+- [x] Define a strategy and high-level description of "the 2i2c pilot" https://github.com/2i2c-org/team-compass/issues/192
+- [ ] Updates to our board process https://github.com/2i2c-org/team-compass/pull/188
+    - [x] More tightly couple the "deliverables meeting" with the "activity board"
+        - The meeting's goal is to put things on the activity board.
+        - Each item should have a person assigned to it
+        - Should use best judgment about workload balance and making sure we're not overextended
```

Comment:
```suggestion
        - Should use best judgment about workload balance and making sure we're not overextended
        - We should make a new board per each sprint, and treat it as immutable as possible
```

Reply:
If we are going into a sprinty way to manage the board, that makes sense...
Some worries about that is the proliferation of boards (unless you can somehow archive them and I think you can close them so that should be enough) and the fact the issues end tied to some specific board that could be useful from retrospective purposes but not that good to a wide project search (modulo that we have per project boards as we have discussed before).

Reply:
IMO these notes are meant to be a reflection of the meeting's conversations and decisions, and I don't remember ever explicitly discussing whether we should use "one board per week, and treat it as immutable". Am I remembering incorrectly? I'm happy to discuss this but it feels like this makes it sound like we discussed and decided, when I don't believe we did.